app/models/group.rb

```diff
@@ -92,6 +92,12 @@ class Group < ActiveRecord::Base # rubocop:disable Metrics/ClassLength
   class_attribute :archival_validation
   self.archival_validation = true
 
+  # If set to true, static and translated sti-class label is used
+  # instead of name attribute value.
+  # name is not indexed for search anymore in that case.
+  class_attribute :static_name
+  self.static_name = false
```

Comment:
das liesse sich in einer Zeile zusammenfassen:
```
class_attribute :static_name, default: false
```

Reply:
ah cool, passe ich noch so an